app/src/main/java/com/epfl/neighborfood/neighborfoodandroid/database/firebase/FirebaseDatabase.java

```diff
@@ -55,15 +47,15 @@ public Task<Void> delete(String collectionPath, String documentPath) {
     }
 
     @Override
-    public Task<String> add(String collectionPath, Map<String, Object> data) {
+    public Task<String> add(String collectionPath, Object data) {
         return database.collection(collectionPath)
                 .add(data).onSuccessTask((documentReference ->
                         Tasks.forResult(documentReference.getId())
                 ));
     }
 
     @Override
-    public Task<DBDataCollection> fetchAll(String collectionPath) {
+    public Task<CollectionSnapshot> fetchAll(String collectionPath) {
         return null;
     }
 }
```

Comment:
Are you planning on implementing the fetchAll function? Maybe add a comment to specify why it's empty for now.